internal/cli/context.go

```diff
@@ -0,0 +1,98 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/10gen/realm-cli/internal/telemetry"
+)
+
+const (
+	appConfigJSON = "config.json"
+)
+
+var (
+	maxDirectoryContainSearchDepth = 8
+)
+
+// Context is the CLI context
+type Context struct {
```

Comment:
[q] I know we use Context in baas, but this gets confusing between the std lib context and our own implementation of context. is there a better name we can use? potentially Settings or even just adding this to the config directly?

Reply:
running out of good names here, I admit.  as it stands, the go std lib Context is not used in this repo right now, so there's no cross-contamination of names there.

this was initially `CommandConfig`, but "config"/"settings" doesn't feel right to me for this data. in my opinion, this data is more contextual than configurable

Reply:
since context is so embedded in the std lib, I would really not like to muddy the waters here. could we just remove this struct altogether and use these fields in line to where we use Context now?